app/modules/theme/provider.tsx

```diff
@@ -85,14 +101,20 @@ function ThemeProvider({
   );
 }
 
-// this is how I make certain we avoid a flash of the wrong theme. If you select
-// a theme, then I'll know what you want in the future and you'll not see this
-// script anymore.
+/** 
+ * This is how I make certain we avoid a flash of the wrong theme. 
```

Comment:
Note: I was not able to replicate this on my end, so please ensure that this theme flash issue does not occur. Thank you!